### DISCUSSION TITLE
"flex-center" utility - @downwindcss/flex-center

### DISCUSSION THREAD

**Discussion by dance2die** [tailwindlabs/tailwindcss#4637] [Show and tell]
Ain't nobody got time to type, `flex justify-center items-center`.

Just type `flex-center` and move on :)

**Demo**: https://codesandbox.io/s/flex-center-demo-v1c3k?file=/public/index.html
**NPM**: https://www.npmjs.com/package/@downwindcss/flex-center
**Source**: https://github.com/downwindcss/flex-center

**Comment by royvanv** [comment]
You wrote a plugin  to basically do `.flex-center { @apply flex justify-center items-center }`?

**Reply by dance2die** [reply]
Yes :wink: 

Been using it for awhile, and found myself needing utils like `flex-center-col/row`, as well (not yet added but will).
